src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -20,6 +20,7 @@ for further details.
 }
 
 .sidebar {
+  background-color: white;
```

Comment:
This fixes an issue where the line above the footer was showing in a Sidebar with little content.